news/2025/2025-10-30-fall-time-mania-mapping-contest.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Fall Time Mania Mapping Contest 2025
+date: 2025-10-30 16:00:00 +0000
+---
+
+The first iteration of FTMC is on it's way with the chilly autumn air! Are you ready to start your holiday mapping?
+
+![](https://i.gyazo.com/9356eb561bd54282db105a324669c887.png)
```

Comment:
To host the news banner, you'll have to:
- Create a new folder in `/wiki/shared/news/...`. The name of which **must** be the same as the news post's. See other folders in the same directory for examples.
- Add the `banner.jpg` and `banner@2x.jpg` to the folder you just made.
- Use the relative path of `banner.jpg` as a banner for the post.